solution/ui/regulations/css/scss/partials/_header.scss

```diff
@@ -308,7 +308,6 @@ header {
         }
 
         .reg-title:after {
-            content: "Search Results";
```

Comment:
was this left over from a long forgotten version of the header?

Reply:
Thats what I was thinking.  If you go to that page it actually doesnt appear anywhere but with my changes it did appear.  Befoer when we did the if part on the django template, the .reg-title was hidden.

Reply:
okay nice.  deleting stale code is always good!